binaries/coordinator/src/run/mod.rs

```diff
@@ -16,6 +16,7 @@ use std::{
     path::PathBuf,
 };
 use uuid::{NoContext, Timestamp, Uuid};
+const DEFAULT_WORKING_DIR: &str = "/tmp";
```

Comment:
I'm not sure this is cross platform. Could we use something like: https://github.com/Stebalien/tempfile ?